tests/core/test_external_search.py

```diff
@@ -4479,7 +4498,9 @@ def test_to_search_document(self):
 
         # Top level keys should be the same
         assert len(result) == len(inapp)
-        assert result[0].keys() == inapp[0].keys()
+
+        # This test is no longer valid with new mapping properties
```

Comment:
@RishiDiwanTT :  if this test is legitimately no longer valid,  it should be removed no?

Reply:
Yes, I seem to have this habit of not deleting old code. Will remove.